cinch/roles/jenkins_slave/tasks/check_swarm_systemd.yml

```diff
@@ -0,0 +1,17 @@
+- name: check if swarm connected to the master with systemd
+  command: systemctl status swarm
+  ignore_errors: yes
+  changed_when: false
+  register: service_swarm_status
+
+- name: grab errors for the last run of the systemd swarm service
+  shell: >
```

Comment:
After reading some systemctl and journalctl docs it was the only way I could find to grab the *last* log output of the swarm service.